java/src/hdf/hdf5lib/H5.java

```diff
@@ -13883,7 +13886,7 @@ public static long H5Tdecode(byte[] buf) throws HDF5LibraryException, NullPointe
         return id;
     }
 
-    private synchronized static native long _H5Tdecode(byte[] buf)
```

Comment:
At first I was going to mention that there should be the corresponding JNI changes, but it seems there is no `_H5Tdecode` implementation currently. @byrnHDF We should either implement that or remove this function from being available.

Reply:
Definitely looks like an oversight! I wonder if it got dropped along the way?

Reply:
Most likely the issue is the byte buffer and translating C memory to java memory